logos/AUTHOR

```diff
@@ -1,16 +1,21 @@
-The MDAnalysis 'Atom' logo was created by Christian Beckstein and is
+The MDAnalysis 'Atom' logo was created by Christian Beckstein
+(@beckstein) in 2011.
 
-Copyright (c) 2011 Christian Beckstein
-
-MDAnalysis Logo 'Atom' by Christian Beckstein is licensed under a
-Creative Commons Attribution-NoDerivs 3.0 Unported License.
-To view a copy of this license, visit
-http://creativecommons.org/licenses/by-nd/3.0/ or send a letter to Creative
-Commons, 444 Castro Street, Suite 900, Mountain View, California, 94041, USA.
+(Prior to 2024-06-21, the MDAnalysis 'Atom' logo was licensed under a
+Creative Commons Attribution-NoDerivs 3.0 Unported License. From this
+date forward the logo and its derivatives may only be used under the
+NumFOCUS trademark guidelines
+https://numfocus.org/trademark-guidelines.)
```

Comment:
Are the parenthesis encapsulating this paragraph necessary here?

Reply:
I added them because this is not a license document and for me they indicated informative text. I wanted to record history somewhere and acknowledge that people may have used the logo previously under the old license. 

LICENSE is what really matters for any situation going forward.